authors/apps/authentication/migrations/0002_user_is_verified.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2 on 2019-05-02 06:32
```

Comment:
@Bakley Kindly commit the initial migrations file instead, as we had agreed as a team

Reply:
@a-braham How do I maintain the initial migration file

Reply:
every time you make changes in the models, push the initial file

Reply:
Okay, thank you @a-braham. I made the changes and pushed the initial migrations files :-)